esphome/components/remote_base/pronto_protocol.cpp

```diff
@@ -107,7 +130,7 @@ void ProntoProtocol::send_pronto_(RemoteTransmitData *dst, const std::vector<uin
   }
 }
 
-void ProntoProtocol::send_pronto_(RemoteTransmitData *dst, const std::string &str) {
+std::vector<uint16_t> decode_pronto(const std::string &str) {
```

Comment:
```suggestion
std::vector<uint16_t> encode_pronto(const std::string &str) {
```

We use `encode` terminology when converting from high-level encoded to low-level timings values.

Reply:
Ok, fixed.